src/main/java/ddingdong/ddingdongBE/domain/form/api/CentralFormApi.java

```diff
@@ -68,17 +68,20 @@ void deleteForm(
     @ResponseStatus(HttpStatus.OK)
     @SecurityRequirement(name = "AccessToken")
     @GetMapping("/my/forms")
-    List<FormListResponse> getAllMyForm(
-            @AuthenticationPrincipal PrincipalDetails principalDetails
-    );
+    List<FormListResponse> getAllMyForm(@AuthenticationPrincipal PrincipalDetails principalDetails);
 
     @Operation(summary = "동아리 지원 폼지 상세조회 API")
     @ApiResponse(responseCode = "200", description = "동아리 지원 폼지 상세조회 성공",
             content = @Content(schema = @Schema(implementation = FormResponse.class)))
     @ResponseStatus(HttpStatus.OK)
     @SecurityRequirement(name = "AccessToken")
     @GetMapping("/my/forms/{formId}")
-    FormResponse getForm(
-            @PathVariable("formId") Long formId
-    );
+    FormResponse getForm(@PathVariable("formId") Long formId);
+
+    @Operation(summary = "동아리 최종 합격 지원자 동아리원 명단 등록API")
+    @ApiResponse(responseCode = "200", description = "최종 합격 지원자 동아리원 명단 등록 성공")
```

Comment:
```suggestion
    @ApiResponse(responseCode = "201", description = "최종 합격 지원자 동아리원 명단 등록 성공")
```
p3)
ResponseStatus랑 상태 맞춰도 좋을 것 같습니다